packages/api/src/services/tags.service.ts

```diff
@@ -13,4 +15,13 @@ export class TagsService {
     async findAll(userId: number): Promise<TransactionTag[]> {
         return this.tagRepository.findBy({ userId });
     }
+
+    async create(req: CreateTagRequest): Promise<TransactionTag> {
+        if (await this.tagRepository.findOneBy(req)) {
+            throw new BadRequestException(ERROR_TAG_NAME_ALREADY_ADDED_TO_ACCOUNT);
+        }
+        const createdTag = this.tagRepository.create(req);
```

Comment:
Do you like this name better? because I do

```suggestion
        const tag = this.tagRepository.create(req);
```